paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormArgumentsFactory.kt

```diff
@@ -20,21 +18,8 @@ internal object FormArgumentsFactory {
             shippingDetails = metadata.shippingDetails,
             billingDetailsCollectionConfiguration = metadata.billingDetailsCollectionConfiguration,
             cbcEligibility = metadata.cbcEligibility,
-        )
-    }
-
-    fun create(
-        paymentMethodCode: PaymentMethodCode,
-        configuration: CustomerSheet.Configuration,
-        merchantName: String,
-        cbcEligibility: CardBrandChoiceEligibility,
-    ): FormArguments {
-        return FormArguments(
-            paymentMethodCode = paymentMethodCode,
-            merchantName = merchantName,
-            billingDetails = configuration.defaultBillingDetails,
-            billingDetailsCollectionConfiguration = configuration.billingDetailsCollectionConfiguration,
-            cbcEligibility = cbcEligibility,
+            hasIntentToSetup = metadata.hasIntentToSetup(),
+            paymentMethodSaveConsentBehavior = metadata.paymentMethodSaveConsentBehavior,
```

Comment:
We used this for `CustomerSheet` but we shouldn't need to since we have `PaymentMethodMetadata` in `CustomerSheet` as well.

Reply:
The form arguments are covered by emulator tests that do full billing details collection.